gtk/PrefsDialog.cc

```diff
@@ -176,6 +176,35 @@ Gtk::Entry* new_entry(tr_quark const key, Glib::RefPtr<Session> const& core)
     return w;
 }
 
+void text_buffer_changed_cb(Glib::RefPtr<Gtk::TextBuffer> buffer, tr_quark const key, Glib::RefPtr<Session> const& core)
+{
+    Gtk::TextBuffer::iterator start, end;
+    buffer->get_bounds(start, end);
+    Glib::ustring value = buffer->get_text(start, end, FALSE);
+
+    core->set_pref(key, value);
```

Comment:
_(nit)_ temporary is only used once, so it could be replaced with

```suggestion
    core->set_pref(key, buffer->get_text(start, end, FALSE));
```

Which might give us some C++-move magic